src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -200,14 +204,27 @@ class LocalBackend extends Backend with LazyLogging {
   private def stageWdlFile(wdlFile: WdlFile, hostInputsPath: Path): WdlFile = {
     val originalPath = Paths.get(wdlFile.value)
     val executionPath = hostInputsPath.resolve(originalPath.getFileName.toString)
+
```

Comment:
Can you put in a comment here describing why this is an issue and how this is solving it?